instances.go

```diff
@@ -242,6 +263,40 @@ func (c *Client) SetDualAuthInstancePolicy(ctx context.Context, enable bool) err
 	return err
 }
 
+// SetRotationInstancePolicy updates the rotation instance policy details associated with an instance.
+// For more information can refet to the Key Protect docs in the link below:
+// will update soon
+func (c *Client) SetRotationInstancePolicy(ctx context.Context, enable bool, intervalMonth *int) error {
+
+	rotationPolicyData := InstancePolicy{
+		PolicyType: RotationPolicy,
+		PolicyData: PolicyData{
+			Enabled: &enable,
+		},
+	}
+
+	if intervalMonth != nil {
```

Comment:
Pls add validation similar to allowed ip policies so that we can get error saying interval_month is required when it's not passed in the request.